lib/Doctrine/ORM/Query.php

```diff
@@ -523,10 +521,8 @@ public function setQueryCacheLifetime($timeToLive): self
 
     /**
      * Retrieves the lifetime of resultset cache.
-     *
-     * @return int
      */
-    public function getQueryCacheLifetime()
+    public function getQueryCacheLifetime(): int
```

Comment:
Not sure why SA isn't complaining here. `queryCacheTTL` prop is uninitializd so I think it may return null

Reply:
I think that's because we are at a dangerously lenient level right now.